framework/console/controllers/FixtureController.php

```diff
@@ -415,7 +415,15 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            // strip $fixturesPath from $fixture's full path
```

Comment:
the added logic should be ensapsulated in a private method with the comments in the description